.github/workflows/database-databricks-integration-test.yml

```diff
@@ -20,8 +20,6 @@ env:
   GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
   DATABRICKS_API_TOKEN: ${{ secrets.DATABRICKS_API_TOKEN }}
-  DATABRICKS_WORKSPACE: https://dbc-f0687849-717f.cloud.databricks.com
```

Comment:
not needed. This was required when trying to boot up a cluster prior to JDBC connection. Now using serverless endpoint, cluster get started in < 5 sec if idle